rest_framework/filters.py

```diff
@@ -294,8 +294,8 @@ def get_template_context(self, request, queryset, view):
             'param': self.ordering_param,
         }
         for key, label in self.get_valid_fields(queryset, view, context):
-            options.append((key, '%s - %s' % (label, _('ascending'))))
-            options.append(('-' + key, '%s - %s' % (label, _('descending'))))
+            options.append((key, f'{label} - {_("ascending")}'))
+            options.append(('-' + key, f'{label} - {_("descending")}'))
```

Comment:
I find the translation-within-and-f-string looks a bit confusing to me.

Perhaps helpful to pull those expressions out the f-string?...

```suggestion
            ascending_translation = _("ascending")
            descending_translation = _("descending")
            options.append((key, f'{label} - {ascending_translation}'))
            options.append(('-' + key, f'{label} - {descending_translation}'))
```